src/Tracker/trackerData.tsx

```diff
@@ -503,4 +464,432 @@ export const analyticsEventsMapping = new Map<
+      properties: {
+        event_type: 'view',
+        document_side: 'front',
+        ui_alerts: { glare: 'warning' },
+      },
+    },
+  ],
+  [
+    'screen_document_back_confirmation_glare_detected',
+    {
+      eventName: 'DOCUMENT_CONFIRMATION_ERROR',
+      properties: {
+        event_type: 'view',
+        document_side: 'back',
+        ui_alerts: { glare: 'warning' },
+      },
+    },
+  ],
+  [
+    'screen_face_selfie_confirmation_no_face_error',
+    {
+      eventName: 'FACE_SELFIE_CONFIRMATION_ERROR',
+      properties: {
+        event_type: 'view',
+        ui_alerts: { no_face: 'error' },
+      },
+    },
+  ],
+  [
+    'screen_face_selfie_confirmation_multiple_faces_error',
+    {
+      eventName: 'FACE_SELFIE_CONFIRMATION_ERROR',
+      properties: {
+        event_type: 'view',
+        ui_alerts: { multiple_faces: 'error' },
+      },
+    },
+  ],
+  [
+    'screen_document_front_confirmation_request_error',
+    {
+      eventName: 'DOCUMENT_CONFIRMATION_ERROR',
+      properties: {
+        event_type: 'view',
+        document_side: 'front',
+        ui_alerts: { request_error: 'error' },
+      },
+    },
+  ],
+  [
+    'screen_document_back_confirmation_request_error',
+    {
+      eventName: 'DOCUMENT_CONFIRMATION_ERROR',
+      properties: {
+        event_type: 'view',
+        document_side: 'back',
+        ui_alerts: { request_error: 'error' },
+      },
+    },
+  ],
+  [
+    'screen_face_face_video_confirmation_request_error',
+    {
+      eventName: 'FACE_VIDEO_CONFIRMATION_ERROR',
+      properties: {
+        event_type: 'view',
+        ui_alerts: { request_error: 'error' },
+      },
+    },
+  ],
+  [
+    'screen_face_selfie_confirmation_request_error',
+    {
+      eventName: 'FACE_SELFIE_CONFIRMATION_ERROR',
+      properties: {
+        event_type: 'view',
+        ui_alerts: { request_error: 'error' },
+      },
+    },
+  ],
+  [
+    'screen_document_front_confirmation_document_detection',
+    {
+      eventName: 'DOCUMENT_CONFIRMATION_ERROR',
+      properties: {
+        event_type: 'view',
+        document_side: 'front',
+        ui_alerts: { document_detection: 'error' },
+      },
+    },
+  ],
+  [
+    'screen_document_back_confirmation_document_detection',
+    {
+      eventName: 'DOCUMENT_CONFIRMATION_ERROR',
+      properties: {
+        event_type: 'view',
+        document_side: 'front',
+        ui_alerts: { document_detection: 'error' },
+      },
+    },
+  ],
+  [
+    'screen_document_front_confirmation_invalid_type',
+    {
+      eventName: 'DOCUMENT_CONFIRMATION_ERROR',
+      properties: {
+        event_type: 'view',
+        document_side: 'front',
+        ui_alerts: { invalid_type: 'error' },
+      },
+    },
+  ],
+  [
+    'screen_document_back_confirmation_invalid_type',
+    {
+      eventName: 'DOCUMENT_CONFIRMATION_ERROR',
+      properties: {
+        event_type: 'view',
+        document_side: 'front',
```

Comment:
This needs to be changed to 'back'
```suggestion
        document_side: 'back',
```